config.sample.php

```diff
@@ -1,39 +1,44 @@
 <?php
 use SiteMaster\Core\Config;
-
 /**********************************************************************************************************************
  * php related settings
  */
-
 ini_set('display_errors', true);
-
 error_reporting(E_ALL);
-
 Config::set('URL', '/'); //Trailing slash is important
-
 /**********************************************************************************************************************
  * DB related settings
  */
-Config::set('DB_HOST'     , 'localhost');
-Config::set('DB_USER'     , 'sitemaster');
-Config::set('DB_PASSWORD' , 'password');
-Config::set('DB_NAME'     , 'sitemaster');
-
+Config::set('DB_HOST', 'localhost');
+Config::set('DB_USER', 'sitemaster');
+Config::set('DB_PASSWORD', 'password');
+Config::set('DB_NAME', 'sitemaster');
 /**********************************************************************************************************************
  * Other settings, including theme
  */
 Config::set('THEME', 'foundation');
-
-
 /**********************************************************************************************************************
  * Plugin related settings
  */
 Config::set('PLUGINS', array(
-    'example' => array('setting'=>'value'),
-    'theme_foundation' => array('setting'=>'value'),
-    'auth_google' => array('setting'=>'value'),
+    'metric_axe' => [
+        // 'weight' => 20, // Adjust weight if desired
+    ],
+    'metric_w3c_html' => [
+        // 'weight' => 20, // Adjust weight if desired
+        // 'service_url' => 'https://validator.unl.edu/', // adjust the service URL if desired
+    ],
```

Comment:
`metric_axe` and `metric_w3c_html` are not bundled with SiteMaster which is causing TravisCI to fail. I think it is helpful to have the examples in this file, but I recommend entirely commenting them out.